text/0027-foreman-plugin-forge.md

```diff
@@ -0,0 +1,141 @@
+- Feature Name: A Forge for Foreman plugins
```

Comment:
Foundry is an excellent name (I spent a total of 120 seconds thinking of a new name, unsuccessfully)